res/css/structures/auth/_SetupEncryptionBody.pcss

```diff
@@ -6,13 +6,6 @@ SPDX-License-Identifier: AGPL-3.0-only OR GPL-3.0-only OR LicenseRef-Element-Com
 Please see LICENSE files in the repository root for full details.
 */
 
-.mx_SetupEncryptionBody_reset {
-    color: $light-fg-color;
-    margin-top: $font-14px;
-
-    .mx_SetupEncryptionBody_reset_link {
-        &.mx_AccessibleButton_kind_link_inline {
-            color: $alert;
-        }
-    }
+.mx_EncryptionCard.mx_SetupEncryptionBody {
+    width: 600px;
 }
```

Comment:
why does this only apply when `mx_SetupEncryptionBody` is inside an `EncryptionCard` and not otherwise?

Also, can you help me understand why this is moving from `mx_CompleteSecurityBody` to here? What is the relationship between `mx_CompleteSecurityBody` and `mx_EncryptionCard.mx_SetupEncryptionBody` ?

Reply:
Hmm.  I'm not sure why I have the `.mx_EncryptionCard` in there.  It looks like it isn't needed -- `mx_SetupEncryptionBody` isn't used anywhere else.

> Also, can you help me understand why this is moving from `mx_CompleteSecurityBody` to here? What is the relationship between `mx_CompleteSecurityBody` and `mx_EncryptionCard.mx_SetupEncryptionBody` ?

`mx_CompleteSecurityBody` is only used when the user initially logs in.  When `SetupEncryptionBody` is used as a modal inside the app, it's just inside of an `.mx_Dialog` (and associated things).  So moving the style here means that it gets used both on initial login and within the app.